pkg/web/templates/partials/apikey/apikey_delete.html

```diff
@@ -1 +1,26 @@
-<div></div>
\ No newline at end of file
+<div class="modal-box">
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-xl">Delete API Key</h1>
+    <button onclick="apikey_modal.close()" class="btn btn-sm btn-circle btn-ghost">
+      <i class="fa-solid fa-x"></i>
+      <span class="sr-only">Close modal</span>
+    </button>
+  </div>
+  <p class="my-4">
+    Are you sure you want to delete the API key? This action cannot be undone.
```

Comment:
```suggestion
    Are you sure you want to revoke the API key? This action cannot be undone.
```

Can we use "revoke" instead of "delete" -- in the future we might not delete the key, but move it to a revoked table instead for auditing purposes.